app/models/document.rb

```diff
@@ -62,6 +63,10 @@ class NotArchiveableError < StandardError; end
     "Site Visit"
   ].freeze
 
+  SITE_NOTICE_TAGS = [
```

Comment:
Had to do similar with Press notice, reckon we can just whack these both in an `OTHER_TAGS` or would you prefer listing it separately